core/app/models/spree/line_item.rb

```diff
@@ -18,7 +18,7 @@ class LineItem < Spree::Base
 
     has_one :product, through: :variant
 
-    has_many :adjustments, as: :adjustable, inverse_of: :adjustable, dependent: :destroy
+    has_many :adjustments, as: :adjustable, inverse_of: :adjustable, dependent: :destroy, autosave: true
```

Comment:
@benjaminwil @jarednorman we missed autosaving the shipment adjustments in this PR, and it's resulting in shipment tax adjustments not getting persisted correctly in Solidus 4.6.0 .... draft PR incoming to address it!